src/elements/icon.js

```diff
@@ -3607,6 +3607,41 @@ class Icon extends React.Component {
+                            {renderGradientColor}
+                        </defs>
+                        <g id="Roster-v1.4" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+                            <g id="Roster-Members-01" transform="translate(-154.000000, -224.000000)">
+                                <g id="70px-Table-+-70px-Table-+-70px-Table-+-70px-Table-Mask" transform="translate(0.000000, 208.000000)">
+                                    <g id="70px-Table">
+                                        <g id="Left-Align-Columns-[0h-t}" transform="translate(11.000000, 0.000000)">
+                                            <g id="Column-1-[0v-r]" transform="translate(142.000000, 0.000000)">
+                                                <g id="smallhandles">
+                                                    {renderGradientColor ? (
+                                                        <mask id={maskId} fill="white">
+                                                            <use xlinkHref={`#${pathId}`} />
+                                                        </mask>
+                                                    ) : null}
+                                                    <use className="icon-use-path" xlinkHref={`#${pathId}`} />
+                                                    {renderGradientColor ? (
+                                                        <rect fill={`url(#${gradientId})`} mask={`url(#${maskId})`} x="0" y="0" width="5px" height="14px" />
+                                                    ) : null}
+                                                </g>
+                                            </g>
+                                        </g>
+                                    </g>
+                                </g>
+                            </g>
+                        </g>
```

Comment:
Is there a need for all these groups (`<g />`) elements? Also, I am a little concerned with the IDs as well with having the multiple icons of the same type on a page.

Reply:
Fixed